Cloudflare/gateway_dns/_meta/smart-descriptions.json

```diff
@@ -9,5 +9,22 @@
         "field": "dns.question.name"
       }
     ]
+  },
+  {
+    "value": "{host.name} ( {user.email} ) requested the resolution of {dns.question.name} from {source.ip}",
+    "conditions": [
+      {
+        "field": "source.ip"
+      },
+      {
+        "field": "host.name"
+      },
+      {
+        "field": "user.email"
+      },
+      {
+        "field": "dns.question.name"
+      }
+    ]
```

Comment:
I suggest moving this smart-description above the previous one, so that the algorithm can use it (the algorithm stop the first matching smart-description).